boards/rfduino/Makefile.features

```diff
@@ -0,0 +1,2 @@
+FEATURES_PROVIDED += cpp
+FEATURES_PROVIDED += periph_uart periph_gpio periph_random periph_rtt periph_cpuid
```

Comment:
@rfswarm Where/To whom did you move it? I don't have any rights for "unknown repository" neither. :smile: